First draft of CCP4I2 GUI

linucks opened the pull request:
This is an initial draft of the CCP4I2 GUI for SIMBAD that supports a few basic objects.

I've created a SimbadResults class in simbad/util/simbad_results the aim of which is to handle the processing of results as this is required by GUI2. The idea is that this could replace a lot of the results handling code that currently resides in (e.g) pyrvapi_results. However, I haven't integrated this yet as I wanted to see what other people thought.

linucks commented:
This updates the pull request to fix the issue with the number of processors, display of the organism drop-down and table display in the CCP4 I2 gui.

hlasimpk commented:
Super happy with how this is working but I do have a few comments!

- I think we need to add in some advanced options that are missing, for example:
`-mr_program <molrep | phaser>` to select the mr program used
`--process_all` to ignore our early termination criteria
`-enan <True | False>` search enantiomorphic space groups

I'm not sure what else is really needed there, maybe have a look through the command line options for anything else that seems useful

And if I'm nitpicking:
- The ccp4i interface is stored in the dir ccp4i whereas ccp4i2 is stored in i2, maybe change to ccp4i2 for consistency
- Change variable names to reference SIMBAD instead of AMPLE

Overall though, it's looking really good

linucks commented:
Ok, I've added the advanced options and changed any AMPLE variable names that I could find.

It doesn't matter where the SIMBAD i2 code is stored as this code will live in the CCP4 devel repository, so we can delete the i2 folder once that's happened.
